src/settings/settings.js

```diff
@@ -30,6 +34,9 @@ document.getElementById('toggle-password-form').addEventListener('change', funct
         form.classList.remove('hidden');
     } else {
         form.classList.add('hidden');
+        localStorage.removeItem('storedHashedPassword')
+        localStorage.removeItem('authenticateValue')
+        localStorage.removeItem('authenticateKey')
```

Comment:
switch to using api file if you create it. - Highly recommended, but if there isn't enough time then we don't have to.